### PR TITLE
Add 01 quickstart

### PR DESCRIPTION
**Pull request by mir-of**
Add English version of basics/01_quickstart.md

### CODE REVIEW

File: en/docs/basics/01_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    (5): ReLU()
+  )
+)
+```
+
+> See in [Build Network](./04_build_network.md)
+
+## Training Model
+
+To train a model, we need a loss function (`loss_fn`) and an optimizer (`optimizer`).The loss function is used to evaluate the difference between the prediction of the neural network and the real label. The optimizer adjusts the parameters of the neural network to make the prediction more and more close to the real label (standard answer). Here, we use [oneflow.optim.SGD](https://oneflow.readthedocs.io/en/master/optim.html?highlight=optim.SGD#oneflow.optim.SGD) to be our optimizer. This process is called back propagation.
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:126]: “标准答案” 在中文里好理解，不过直译成英文 standard answer 不一定是原来的意思，是不是用 benchmark 会好一点？
这个请董文文王开妍确认下。 @wwd190906 @BONNNIE

File: en/docs/basics/01_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+```python
+loss_fn = nn.CrossEntropyLoss()
+optimizer = flow.optim.SGD(model.parameters(), lr=1e-3)
+```
+
+A train function is defined for training. In a single training loop, the model make forward propagation, calculating loss, and back propagation to update model parameters.
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:133]:
```suggestion
The `train` function is defined for training. In a single training loop, the model make forward propagation, calculating loss, and back propagation to update model parameters.
```
这里的 `train` 就是指的下面代码的 `train` 函数

File: en/docs/basics/01_quickstart.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+        optimizer.zero_grad()
+        loss.backward()
+        optimizer.step()
+
+        current = batch * BATCH_SIZE
+        if batch % 100 == 0:
+            print(f"loss: {loss:>7f}  [{current:>5d}/{size:>5d}]")
+```
+
+We also define a test function to verify the accuracy of the model:
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:154]:
```suggestion
We also define a `test` function to verify the accuracy of the model:
```

File: en/docs/basics/01_quickstart.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+            test_loss += loss_fn(pred, y)
+            bool_value = (pred.argmax(1).to(dtype=flow.int64)==y)
+            correct += float(bool_value.sum().numpy())
+    test_loss /= num_batches
+    print("test_loss", test_loss, "num_batches ", num_batches)
+    correct /= size
+    print(f"Test Error: \n Accuracy: {(100*correct):>0.1f}, Avg loss: {test_loss:>8f}")
+```
+
+Now can we use the train function to begin the train process for several epochs and then use test function to assess the accuracy of the network at the end of each epoch:
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:175]:
```suggestion
Now we can use the `train` function to begin the train process for several epochs and then use `test` function to assess the accuracy of the network at the end of each epoch:
```

File: en/docs/basics/01_quickstart.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+-------------------------------
+loss: 2.288640
+loss: 2.286367
+...
+```
+> See in [Autograd](./05_autograd.md), [Backpropagation and Optimizer](./06_optimization.md)
+
+## Saving and Loading Model
+
+Use [oneflow.save](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.save#oneflow.save) to saves the model. The saved model can be then loaded by [oneflow.load](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.load#oneflow.load) to re-creating the model structure and make predictions on it.
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:209]:
```suggestion
Use [oneflow.save](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.save#oneflow.save) to save the model. The saved model can be then loaded by [oneflow.load](https://oneflow.readthedocs.io/en/master/oneflow.html?highlight=oneflow.load#oneflow.load) to make predictions on it.
```

`re-creating the model` 是不是不够准确，模型的形状是要提前已经编写好，然后 `oneflo.load` 只是把参数加载进去。并没有“创建”模型的这个功能？

File: en/docs/basics/01_quickstart.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+
+```python
+flow.save(model.state_dict(), "./model")
+```
+> See in [Model Load and Save](./07_model_load_save.md)
+
+## QQ Group
+Any problems encountered during the install or usage, welcome to join the QQ Group to discuss with OneFlow developer and enthusiasts:
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:217]: 这个可以讨论下，是否用QQ（让老外用QQ，等于是不为他们提供交流机会）。

我们之前是留学生 LiuQi 的建议下，用的是 discord，不过貌似没坚持下来，我们可以讨论下为什么……（我觉得一个原因是，discord的 APP 版本很难设置翻墙，被迫只能用 web 版本，而 web 版本并不方便）

File: en/docs/basics/01_quickstart.md [hunk 7]
```diff
@@ -0,0 +1,221 @@
+# QUICKSTART
```
[the remark this answers is not in the document]
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:1]: 因为这一行是标题，所以大写

File: en/docs/basics/01_quickstart.md [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+import oneflow.nn as nn
+import oneflow.utils.vision.transforms as transforms
+
+BATCH_SIZE=128
+```
+
+
+## Working with Data
+
+OneFlow has two primitives to work with data, which are Dataset and Dataloader.
```
[the remark this answers is not in the document]
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:18]: 对，这里我专门删掉了补到了本小节结束

File: en/docs/basics/01_quickstart.md (hunk 6 above)
Comment (doombeaker) [reply en/docs/basics/01_quickstart.md:217]: 我想了下，是不是这里放 Oneflow-Inc/oneflow 仓库的 issues 链接比较好？

Comment (mir-of) [reply en/docs/basics/01_quickstart.md:217]: 我们的论坛有英文版块吗？可不可以用？

File: en/docs/basics/01_quickstart.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:1]: 这里我改成小写了

File: en/docs/basics/01_quickstart.md [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+```
+
+Out:
+
+```text
+x.shape: flow.Size([128, 1, 28, 28])
+y.shape: flow.Size([128])
+```
+
+> see in [Dataset and Dataloader](./03_dataset_dataloader.md)
```
[the remark this answers is not in the document]
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:76]: 这里是因为不知情删掉了，已改回按钮式

File: en/docs/basics/01_quickstart.md [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+
+BATCH_SIZE=128
+```
+
+
+## Working with Data
+
+OneFlow has two primitives to work with data, which are Dataset and Dataloader.
+
+The [oneflow.utils.vision.datasets](https://oneflow.readthedocs.io/en/master/utils.html#module-oneflow.utils.vision.datasets) module contains a number of real data sets (such as MNIST, CIFAR 10, FashionMNIST) .
```
[the remark this answers is not in the document]
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:20]: 已修改

File: en/docs/basics/01_quickstart.md [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+
+We can use `oneflow.utils.vision.datasets.MNIST` to get the training set and test set data of MNIST
```
[the remark this answers is not in the document]
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:22]: 已修改

File: en/docs/basics/01_quickstart.md (hunk 2 above)
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:133]: 已修改

File: en/docs/basics/01_quickstart.md (hunk 3 above)
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:154]: 已修改

File: en/docs/basics/01_quickstart.md (hunk 4 above)
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:175]: 已修改

File: en/docs/basics/01_quickstart.md (hunk 5 above)
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:209]: 已修改

File: en/docs/basics/01_quickstart.md (hunk 6 above)
Comment (mir-of) [reply en/docs/basics/01_quickstart.md:217]: 这个待定，还未修改

Comment (doombeaker) [reply en/docs/basics/01_quickstart.md:220]: 问了许昌华，OneFlow 的论坛做了一半没有推动了。所以英文版没有，其实中文版也没有。

这个先合并到 revision_en_version 上线前我在统一改

File: en/docs/basics/01_quickstart.md [hunk 12]
```diff
@@ -0,0 +1,224 @@
...
+  )
+)
+```
+
+> [:link: Build Network](./04_build_network.md){ .md-button .md-button--primary}
+
+
+## Training Model
+
+To train a model, we need a loss function (`loss_fn`) and an optimizer (`optimizer`).The loss function is used to evaluate the difference between the prediction of the neural network and the real label. The optimizer adjusts the parameters of the neural network to make the prediction more and more close to the real label (standard answer). Here, we use [oneflow.optim.SGD](https://oneflow.readthedocs.io/en/master/optim.html?highlight=optim.SGD#oneflow.optim.SGD) to be our optimizer. This process is called back propagation.
```
Comment (doombeaker) [inline en/docs/basics/01_quickstart.md:128]:
```suggestion
To train a model, we need a loss function (`loss_fn`) and an optimizer (`optimizer`).The loss function is used to evaluate the difference between the prediction of the neural network and the real label. The optimizer adjusts the parameters of the neural network to make the prediction more and more close to the real label (expected answer). Here, we use [oneflow.optim.SGD](https://oneflow.readthedocs.io/en/master/optim.html?highlight=optim.SGD#oneflow.optim.SGD) to be our optimizer. This process is called back propagation.
```

**Review by doombeaker — APPROVED** [review]
